p2p/switch.go

```diff
@@ -30,12 +31,22 @@ const (
 	reconnectBackOffBaseSeconds = 3
 )
 
+// The following constants modify the number of events sent to a metric based on severity
+const (
+	Fatal = iota
```

Comment:
These could use a prefix for clarity when referred to. Maybe `PeerMarkFatal` etc.

Reply:
Agreed.